camaya-softdev/ordering-kiosk
# Feat/special instruction

**What changed**

- Added special instruction to the view order page and receipt page

**Implemented Solution**

- Add a textarea to the view order page
- Match the design to the prototype
- Create a state to store it in Redux
- Access it and display to the receipt component
- Pass it in the "formData" then modify the backend controller to insert the data into database

**How can it be tested**

Frontend page
- Open ordering kiosk
- login the credentials
- Order a product
- click the view orders
- check if the special instruction exists
- insert a message
- then proceed to checkout
- check the receipt if the message is displaying
- then open the admin ordering kiosk
- check the order
- check the remarks column if the message is displaying

Backend
- Open admin ordering kiosk
- Login the credentials
- You will automatically directed to order dashboard
- Check the column "special instruction" if it is existed
